liquidhaskell-boot/src/Language/Haskell/Liquid/Bare/DataType.hs

```diff
@@ -393,15 +394,19 @@ makeConTypes myName env specs =
 
 makeConTypes' :: ModName -> Bare.Env -> (ModName, Ms.BareSpec)
```

Comment:
Maybe add some comments to `makeConTypes'` and `makeConTypes''`.

Reply:
4ea275ef1ecbc4b183aea5fb203684d8a9353813